lib/Core/Executor.h

```diff
@@ -101,7 +101,24 @@ class Executor : public Interpreter {
 
   typedef std::pair<ExecutionState*,ExecutionState*> StatePair;
 
+  enum TerminateReason {
+    Abort,
+    Assert,
+    Exec,
+    External,
+    Free,
+    Model,
+    Overflow,
+    Ptr,
+    ReadOnly,
+    ReportError,
+    User,
+    Unhandled,
```

Comment:
ISO C/C++ conformance. If you enable the 'pedantic' warnings you can see it. Not a big deal by the way.